src/views/MoviesPage.vue

```diff
@@ -41,11 +41,18 @@ const selectOptions = computed(() => {
   </div>
   <div class="controls">
     <BaseInput v-model="query" input-name="search">SEARCH</BaseInput>
-    <BaseSelect v-model="category" :options="selectOptions">CATEGORY</BaseSelect>
+    <BaseSelect v-model="category" :options="selectOptions"
+      >CATEGORY</BaseSelect
```

Comment:
Small tip 💡  - Whenever we have something typed in in FULL CAPS - it's actually a better practice to type it normally, but use CSS to change it to be displayed as such - by using the `text-transform: uppercase`.
This way we may write our body copy however we feel like is best to us - but leave the presentation to CSS.

Reply:
I am pretty sure I have defined label element as `text-transform: uppercase` just by force of habit provided text value in this format as well. I will take the tip :)